### PR TITLE
corkscrew: replace WEBrick with TCPServer for test, update license

### PR DESCRIPTION
**Pull request by timsutton**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----
* replace WEBrick with TCPServer (as suggested [here](https://github.com/Homebrew/homebrew-core/pull/162041#issuecomment-1937013129))
* update license to GPL 2.0 to GPL-2.0-or-later

### CODE REVIEW

File: Formula/c/corkscrew.rb [hunk 1]
```diff
@@ -3,7 +3,7 @@ class Corkscrew < Formula
   homepage "https://packages.debian.org/sid/corkscrew"
   url "https://deb.debian.org/debian/pool/main/c/corkscrew/corkscrew_2.0.orig.tar.gz"
   sha256 "0d0fcbb41cba4a81c4ab494459472086f377f9edb78a2e2238ed19b58956b0be"
-  license "GPL-2.0"
+  license "GPL-2.0-or-later"
```
Comment (timsutton) [inline Formula/c/corkscrew.rb:6]: Basing this on https://github.com/patpadgett/corkscrew?tab=GPL-2.0-1-ov-file

**Comment by timsutton** [comment]
Relates to https://github.com/Homebrew/homebrew-core/pull/162041#issuecomment-1937013129

**Comment by bevanjkay** [comment]
@timsutton I'm not sure we should be only running syntax here to make sure the tests pass on CI. But can add `ci-no-bottles`.

**Comment by timsutton** [comment]
@bevanjkay Ah, thank you for pointing that out! I didn't realize the label would cause it to skip tests, but I guess that makes sense. I'll remove the label then.

**Review by p-linnane — APPROVED** [review]

**Pull request merged**